### DISCUSSION TITLE
🚀 Introducing mWhisper-Flamingo, a multilingual audio-visual speech recognition model!

### DISCUSSION THREAD

**Discussion by roudimit** [openai/whisper#2519] [Show and tell]
Excited to introduce mWhisper-Flamingo! Check out the demo video; mWhisper-Flamingo can transcribe multilingual speech with heavy background noise!

- Paper: [ArXiv](https://arxiv.org/abs/2502.01547)
- Code, pre-trained models, Notebook: [GitHub](https://github.com/roudimit/whisper-flamingo)
- 1m demo of Whisper-Flamingo (same video below): [YouTube link](https://youtu.be/Z6VYvlaErTo)

https://github.com/user-attachments/assets/31ef2477-edd0-4dd1-9189-2b74e433d98a

mWhisper-Flamingo is the multilingual follow-up to Whisper-Flamingo which converts Whisper into an AVSR model (but was only trained/tested on English videos).
We trained mWhisper-Flamingo on videos in 9 languages, and it outperforms audio-only Whisper on noisy audio!
Whisper-Flamingo's default training setup yielded minor improvements in noisy multilingual WER, despite significant improvements for English.
To fix this, we introduce decoder modality dropout by training the model both on paired audio-visual inputs and separate inputs.
We are releasing our audio-visual models in two sizes (Medium, Small), as well as the audio-only models fine-tuned on noisy audio.

Let me know if you have any comments or questions!
